src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const rewers = {};
+
+  for (const key in robot) {
+    rewers[robot[key]] = key;
+  }
+
+  if (Object.keys(rewers).length !== Object.keys(robot).length) {
```

Comment:
```suggestion
  if (robot[key] in rewers)
```
Можна набагато простіше